crates/compiler/gen_dev/src/lib.rs

```diff
@@ -678,6 +694,13 @@ trait Backend<'a> {
                 self.load_literal_symbols(args);
                 self.build_fn_call(sym, fn_name, args, arg_layouts, ret_layout)
             }
+            Symbol::NUM_DIV_TRUNC | Symbol::NUM_DIV_FRAC => self.build_run_low_level(
```

Comment:
No diff betwwen DIV_TRUNC and DIV_FRAC? Is DIV_FRAC what is used for floats and decimal types?

Reply:
Yes they both use the same lowlevel. But actually I don't love that, so I'll make special lowlevels for these

Reply:
fixed this now by adding an extra lowlevel. Turns out `/` is not actually used for integers, which is convenient.